File: IAL-002/Listas/4-Funções/03.py
def potencia(b, e):
    # Recebe: um inteiro b (b>0) e um inteiro e (e>=0).
    # Retorna: um inteiro com a potencia de b elevado a e.
    # Obs: nao usar o operador de potenciacao ou qualquer outra funcao
    pot = 1
    for i in range(e):
        pot = pot*b
    return pot

File: ILP-010/Listas/1-Funcoes/03.c
int potencia(float b, int e){
//Recebe: um real (b > 0.0) e um inteiro e (e >= 0).
//Retorna: a potencia de b elevado a e.
  int i;
  float pot=b;
  for(i=1; i<e; i++){
    pot*=b;
  }
  return pot;
}